P5/Exemplars/tei_jtei.odd

```diff
@@ -2525,19 +2534,17 @@
                   </sch:report>
                 </sch:rule>
                 <sch:rule context="tei:note" role="warning">
-                  <sch:report test="not(@rend='inside.parenthesis') and following::text()[not(ancestor::tei:note)][1][matches(., '^\)')]" sqf:fix="note.rend.add">
+                  <sch:report test="following::text()[not(ancestor::tei:note)][1][matches(., '^\)')]">
```

Comment:
Since you're checking for the beginning of the string, does it make sense to normalize-space it first?

Reply:
Sure, thanks. I'll change that.